data/useCvState.ts

```diff
@@ -14,6 +14,7 @@ import type {
 const state = reactive({
   formSettings: { ...cvSettingsEmptyTemplate } as Cv,
   isLoading: true,
+  profileImageData: null,
```

Comment:
This should be a property of formSettings object because, otherwise, the profile image data will be lost when downloading/uploading the CV in JSON format.

Reply:
My original intention is to simply add a feature for uploading profile image, didn't consider how to persist. Even though move profileImageData to formSettings it can not be persisted directly, because the image is a file data, cannot be stored in downloaded JSON file, situation is similar that it cannot be simply restored when upload a JSON file only.

So, to persist the profile image data I think it requires some design and works which are out of the feature of 'upload profile image' itself, e.g. maybe "download CV in JSON format" will be replaced with "download CV data in a zip file(which contains both JSON file and profile image)", and it probably need a default profile image to be shipped with the app for the first loading, so several things may need be taken into consideration.

So, maybe we can add the 'upload profile image' feature as is and put the 'persist profile image data' into another enhancement later.

Reply:
Thank you, I see your point now! 

However, there're ways to store an image in JSON such as using Base64 encoding and Data URIs. I think, for this case, using DataURIs and resizing in the client is a feasible solution while keeping it simple and everything in one file. But as you said, we could see later about the .zip file. 

See one example here: https://img.ly/blog/how-to-resize-an-image-with-javascript/

I am going to commit these changes in this PR for you to check. 

Thank you again for all your work and suggestions 🙏🏻